generators/app/templates/resource/src/index.js

```diff
@@ -0,0 +1,125 @@
+// import React from 'react';
```

Comment:
Why is this commented out? I get tests, but this seem unnecessary

Reply:
This is an example / skeleton. Makes it easy to start by just uncommenting it.